tests/behat/features/bootstrap/FeatureContext.php

```diff
@@ -38,5 +40,9 @@ public function gatherContexts(BeforeScenarioScope $scope)
         $this->minkContext = $environment->getContext(MinkContext::class);
         // Set the testcookie, effectively putting the Stepup suite in test mode
         $this->minkContext->getSession()->setCookie('testcookie', 'testcookie');
+
+        $this->minkContext->getSession('ra')->setCookie('testcookie', 'testcookie');
+        $this->minkContext->getSession('ra')->setCookie('testcookie', 'testcookie');
+        $this->minkContext->getSession('ra')->setCookie('testcookie', 'testcookie');
```

Comment:
Your intention was to set the testcookie on all sessions, `ra` was only targetted by this change.

Reply:
I havn't done anyything with this to prevent conflicts